conans/client/manager.py

```diff
@@ -177,22 +178,18 @@ def export(self, user, channel, conan_file_path, keep_source=False, filename=Non
         export_conanfile(output, self._client_cache, conanfile, src_folder, conan_ref, keep_source,
                          filename)
 
-    def package_files(self, reference, source_folder, build_folder, package_folder, profile,
-                      force):
-        """ Bundle pre-existing binaries
-        @param reference: ConanFileReference
-        """
+    def export_pkg(self, reference, source_folder, build_folder, profile, force):
```

Comment:
Always calling package()

Reply:
Then probably we want ``conan new --bare`` to have a ``package()`` method with ``self.copy("*")`` by default.